pyproject.toml

```diff
@@ -84,5 +55,49 @@ version-header = "="
 version-strings = ["setup.py"]
 build-system = "setuptools"
 
+[tool.pdm]
+
+[tool.pdm.scripts]
+docbuild = "invoke docbuild"
+docserve = "invoke docserve"
+
+[tool.pdm.dev-dependencies]
+dev = [
+    "BeautifulSoup4<5.0,>=4.9",
+    "jinja2~=3.1.2",
+    "lxml<5.0,>=4.3",
+    "markdown~=3.4.3",
+    "typogrify<3.0,>=2.0",
+    "sphinx<6.0,>=5.1",
+    "furo==2023.03.27",
+    "livereload<3.0,>=2.6",
+    "psutil<6.0,>=5.7",
+    "pygments~=2.15",
+    "pytest<8.0,>=7.1",
+    "pytest-cov<5.0,>=4.0",
+    "pytest-sugar<1.0.0,>=0.9.5",
+    "pytest-xdist<3.0,>=2.0",
+    "tox<4.0,>=3.13",
+    "flake8<4.0,>=3.8",
+    "flake8-import-order<1.0.0,>=0.18.1",
+    "invoke<3.0,>=2.0",
+    "isort<6.0,>=5.2",
+    "black<20.0,>=19.10b0",
+]
+
+[tool.pdm.build]
+includes = []
```

Comment:
Is this `[tool.pdm.build]` section required? I don't see it listed in the relevant docs: https://pdm.fming.dev/latest/reference/build/

Reply:
It was just the start to include additional files that are mentioned in the `Manifest.in`. But these files don't exist in the existing release anyhow, so I removed it again.